Headers/DebugServer2/Architecture/ARM/HardwareBreakpointManager.h

```diff
@@ -23,25 +23,25 @@ class HardwareBreakpointManager : public BreakpointManager {
   ~HardwareBreakpointManager() override;
 
 public:
-  virtual ErrorCode add(Address const &address, Type type, size_t size,
-                        Mode mode) override {
+  ErrorCode add(Address const &address, Type type, size_t size,
```

Comment:
Yep that's good. I'm just saying the commit message is a bit confusing. The patch itself is something we want.